### PR TITLE
Add Spanish translation (complete up to Day 1: Afternoon)

### PR DESCRIPTION
**Pull request by wfranck**
Part of #284.

### CODE REVIEW

**Comment by mgeisler** [comment]
Nice, thanks for putting this up! I see you've already translated 500 strings, amazing!

The build failure is because of some double-quotes being translated into nice curly quotes in the Rust code snippets. Let's fix this and merge it :smile:

**Comment by wfranck** [comment]
> Nice, thanks for putting this up! I see you've already translated 500 strings, amazing!
> 
> The build failure is because of some double-quotes being translated into nice curly quotes in the Rust code snippets. Let's fix this and merge it 😄

I'll have to see how work test. I copy and paste original English code into the translation and it fails.

**Comment by wfranck** [comment]
I'll have to see how work test. I copy and paste original English code into the translation and it fails.

**Comment by wfranck** [comment]
I'll have to see how tests works. I copy and paste original English code into the translation and it fails.

**Comment by wfranck** [comment]
I need help to know how to run tests locally. It's a grammar correction error from Poedit.
Until I know how to test I leave it on pause.
`mdbook test` only runs the tests in English and I don't know how to do it pointing to es.po

**Comment by mgeisler** [comment]
> Until I know how to test I leave it on pause.
> `mdbook test` only runs the tests in English and I don't know how to do it pointing to es.po

```sh
$ MDBOOK_BOOK__LANGUAGE=es mdbook test
```

should do the trick. I got this from the failing check on the PR.

We should of course document this in TRANSLATIONS.md if we haven't already.

File: po/es.po [hunk 1]
```diff
@@ -0,0 +1,14084 @@
...
+"include the upper bound.\n"
+"* The `match` expression in `fizzbuzz()` is doing a lot of work. It is expanded below to show what "
+"is happening."
+msgstr ""
+"* En `main` invocamos a una función escrita debajo. No es necesario hacer declaraciones para "
+"referenciar.\n"
+"* La declaración de parámetros es por tipo (lo contrario a otros lenguajes), y devuelven tipo.\n"
+"* La última expresión en una función (u otro bloque) es el tipo que devuelve. Y omite el `;` al "
+"final.\n"
+"* Algunas funciones no devuelven valor y devuelven el `tipo unit`, `()`. El compilador inferir `-"
```
Comment (kevinislas2) [inline po/es.po:2877]:
```suggestion
"* Algunas funciones no devuelven valor y devuelven el `tipo unit`, `()`. El compilador va a inferir `-"
```

**Review by kevinislas2 — APPROVED** [review]
I reviewed a bit past Day 1: Afternoon. Sorry! feel free to ignore all suggestions past Day 1: Afternoon

**Comment by kevinislas2** [comment]
Thanks for the PR! Left some comments and minor suggestions here and there.

File: po/es.po [hunk 2]
```diff
@@ -0,0 +1,14084 @@
...
+"Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.2.2\n"
+
+#: src/SUMMARY.md:3
+msgid "Welcome to Comprehensive Rust 🦀"
+msgstr "Bienvenido a Comprehensive Rust 🦀"
+
+#: src/SUMMARY.md:4
+msgid "Running the Course"
+msgstr "Seguir el Curso"
```
[the remark this answers is not in the document]
Comment (wfranck) [reply po/es.po:21]: I take this context "Running the Course" is related to 'follow a guide'. Like 'follow the advice' not 'continue the advice'.

File: po/es.po [hunk 3]
```diff
@@ -0,0 +1,14084 @@
...
+msgid "Keyboard Shortcuts"
+msgstr "Accesos Rápidos del Teclado"
+
+#: src/SUMMARY.md:7
+msgid "Translations"
+msgstr "Traducciones"
+
+#: src/SUMMARY.md:8
+msgid "Using Cargo"
+msgstr "Usando Cargo"
```
[the remark this answers is not in the document]
Comment (wfranck) [reply po/es.po:37]: there it didn't say 'How to use Cargo'

File: po/es.po [hunk 4]
```diff
@@ -0,0 +1,14084 @@
...
+msgid "Rust Ecosystem"
+msgstr "Ecosistea Rust"
+
+#: src/SUMMARY.md:10
+msgid "Code Samples"
+msgstr "Ejemplos de Código"
+
+#: src/SUMMARY.md:11
+msgid "Running Cargo Locally"
+msgstr "Correr Cargo Localmente"
```
[the remark this answers is not in the document]
Comment (wfranck) [reply po/es.po:49]: is the same meaning: run, use, launch... apps

**Comment by antodellanzo** [comment]
Hi @wfranck! We have done an automatic translation for Spanish, and the idea is to go through each automatically-translated item and check if it's right or fix it. I have created a branch with the [automatic translation](https://github.com/google/comprehensive-rust/commit/afe21b058282204395394b68a0a355f457d459f9). It might be helpful for you to take a look (in case you are translating each phrase manually), so you can speed up your process. I haven't applied your current changes to this file I'm attaching here, so please ignore everything before Day 1 as it hasn't been reviewed/fixed.

**Comment by wfranck** [comment]
Thank you @antodellanzo . I didn't take the original translated file because the automatic translations are not quite correct, sometimes it gets confusing and some programming terms are easier when they are not translated.
I'm not going to take a branch created one hour ago and start from scratch after my commit on February 27th.
So I prefer to continue reading the course in English without doing the translation or working and have this in my own repository.
I'm sure someone will be able to translate it better. So I leave this PR.

**Pull request closed without merging**